resources/definitions/ultimaker_s8.def.json

```diff
@@ -389,6 +408,13 @@
         "machine_nozzle_cool_down_speed": { "default_value": 1.3 },
         "machine_nozzle_heat_up_speed": { "default_value": 0.6 },
         "machine_start_gcode": { "default_value": "M213 U0.1 ;undercut 0.1mm" },
+        "material_bed_temperature":
+        {
+            "force_depends_on_settings": [
+                "support_extruder_nr",
+                "support_enable"
```

Comment:
Is there a specific need for this ?

Reply:
Yes, the UI would not update correctly for these new formula's:
build_volume_temperature = =50 if extruders_enabled_count > 1 and (not support_enable or (extruder_nr != support_extruder_nr)) else 24